src/streamlink/plugins/ceskatelevize.py

```diff
@@ -5,131 +5,168 @@
-            },
-            headers={
-                "x-addr": "127.0.0.1",
+                "data": json.dumps(
+                    {
+                        "contentType": "live",
+                        "items": [{
+                            "id": id,
+                            "key": f"{key}",
+                            "assetId": "CT4DRM",
+                            "playerType": "dash",
+                            "date": f"{date}",
+                            "requestSource": "front-sport",
+                            "drm": 0,
+                            "quality": "web",
+                        }],
+                    },
+                    separators=(',', ':'),
+                ),
             },
             schema=validate.Schema(
                 validate.parse_json(),
                 {
-                    validate.optional("streamingProtocol"): str,
-                    "url": validate.any(
-                        validate.url(),
-                        "Error",
-                        "error_region"
-                    )
-                }
+                    "RESULT": self.schema_playlist,
+                },
+                validate.get(("RESULT", "playlist", 0, "streamUrls", "main")),
             ),
         )
 
-        if data["url"] in ["Error", "error_region"]:
-            log.error("This stream is not available")
-            return
-
-        url = self.session.http.get(
-            data["url"],
-            schema=validate.Schema(
-                validate.parse_json(),
-                {
-                    "playlist": [{
-                        validate.optional("type"): str,
-                        "streamUrls": {
-                            "main": validate.url(),
-                        }
-                    }]
+    def get_channel(self):
+        schema_data = validate.Schema(
+            validate.parse_html(),
+            validate.xml_xpath_string(".//script[@id='__NEXT_DATA__'][text()]/text()"),
+            str,
+            validate.parse_json(),
+            {
+                "props": {
+                    "pageProps": {
+                        "data": {
+                            "liveBroadcast": {
+                                "id": str,
+                                "current": {
+                                    "channelName": str,
+                                },
+                            },
+                        },
+                    },
                 },
-                validate.get(("playlist", 0, "streamUrls", "main"))
-            )
+            },
+            validate.get(("props", "pageProps", "data", "liveBroadcast")),
+            validate.union_get("id", ("current", "channelName")),
         )
-        return DASHStream.parse_manifest(self.session, url)
+
+        try:
+            id, channel_name = self.session.http.get(self.url, schema=schema_data)
+        except PluginError:
+            return
```

Comment:
I know you're just continuing the pattern from before, but it's probably not a good idea to suppress validation errors (PluginError) unless there is some fallback mechanic implemented, which there is none of that here.

Reply:
@mkbloke Could you please quickly fix this once you have the time, so we can merge this?